readme.md

```diff
@@ -107,6 +143,50 @@ Returns a `Promise` for the calculated file hash.
 
 Returns the calculated file hash.
 
+### hasha.createThread()
+
+Requires Node.js v10.5.0 or later.<br>
+Returns a new worker thread.
+
+### hasha.thread
+
+Requires Node.js v10.5.0 or later.<br>
+Lazy loaded worker thread.
+
+## Worker thread API
+
+##### thread.worker
+
+Type: [Worker](https://nodejs.org/api/worker_threads.html#worker_threads_class_worker)<br>
+
+### thread.fromString(input, [options])
+
+Returns a `Promise` for the calculated hash.
+
+### thread.stream([options])
+
+Returns a [hash transform stream](https://nodejs.org/api/crypto.html#crypto_class_hash).
+
+### thread.fromStream(stream, [options])
+
+*Runs on the main thread.*<br>
+Returns a `Promise` for the calculated hash.
+
+### thread.fromFile(filepath, [options])
+
+Returns a `Promise` for the calculated file hash.
+
+### thread.fromFileSync(filepath, [options])
+
+Returns a `Promise` for the calculated file hash.
+
+### thread.[terminate](https://nodejs.org/api/worker_threads.html#worker_threads_worker_terminate)\()
+
+Stop all JavaScript execution in the worker thread as soon as possible
+
+### thread.[on](https://nodejs.org/api/events.html#events_emitter_on_eventname_listener)\(event, listener)
+
+Adds a new event listener to the worker thread. See [Events](https://nodejs.org/api/worker_threads.html#worker_threads_worker_terminate).
```

Comment:
Wrong URL, I meant for it to be: https://nodejs.org/api/worker_threads.html#worker_threads_event_error